### PR TITLE
dev/v3.0: fix resource configuration

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add the resource configuration requirements for NUMA architecture CPU

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1704

### Which version does your change affect? 
dev, v3.0

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 1]
```diff
@@ -125,9 +125,11 @@ TiDB Operator uses `Helm` to deploy and manage TiDB clusters. The configuration
 
 ## Resource configuration
 
-Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. `requests` and `limits` are the resource configuration items listed in the table above. They respectively refer to the minimum requirement for and maximum limit on resources. `limits` must be greater than or equal to `request`. It is recommended to set `limits` to be equal to `requests`, which ensures that the service achieves Guaranteed-level QoS.
+Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. PD, TiKV and TiDB are the core service components of a TiDB cluster. In a production environment, their resource configuration must be specified manually according to component needs. Detailed reference: [Hardware Recommendations](/how-to/deploy/hardware-recommendations.md).
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]:
```suggestion
Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. PD, TiKV and TiDB are the core service components of a TiDB cluster. In a production environment, their resource configurations must be specified according to component needs. Detailed reference: [Hardware Recommendations](/how-to/deploy/hardware-recommendations.md).
```

File: v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 2]
```diff
@@ -125,9 +125,11 @@ TiDB Operator uses `Helm` to deploy and manage TiDB clusters. The configuration
 
 ## Resource configuration
 
-Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. `requests` and `limits` are the resource configuration items listed in the table above. They respectively refer to the minimum requirement for and maximum limit on resources. `limits` must be greater than or equal to `request`. It is recommended to set `limits` to be equal to `requests`, which ensures that the service achieves Guaranteed-level QoS.
+Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. PD, TiKV and TiDB are the core service components of a TiDB cluster. In a production environment, their resource configuration must be specified manually according to component needs. Detailed reference: [Hardware Recommendations](/how-to/deploy/hardware-recommendations.md).
```
Comment (TomShawn) [inline v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]:
```suggestion
Before deploying a TiDB cluster, it is necessary to configure the resources for each component of the cluster depending on your needs. PD, TiKV and TiDB are the core service components of a TiDB cluster. In a production environment, their resource configurations must be specified according to component needs. Detailed reference: [Hardware Recommendations](/how-to/deploy/hardware-recommendations.md).
```

**Comment by juliezhang1112** [comment]
PTAL :) @xiaojingchen

File: dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md [hunk 3]
```diff
@@ -125,9 +125,11 @@ TiDB Operator uses `Helm` to deploy and manage TiDB clusters. The configuration
...
 
-PD, TiKV and TiDB are the core service components of a TiDB cluster. In a production environment, their resource configuration must be specified manually according to component needs. Detailed reference: [Hardware Recommendations](/how-to/deploy/hardware-recommendations.md). In a testing environment, you can directly use the default configuration in the `values.yaml` file without any manual setting.
+To ensure the proper scheduling and stable operation of the components of the TiDB cluster in Kubernetes, it is recommended to set Guaranteed-level QoS by letting `limits` equal to `requests` when configuring resources. Detailed reference: [Configure Quality of Service for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/)。
+
+If you are using a NUMA-based CPU, you need to enable `Static`'s CPU management policy on the node for better performance. In order to allow the TiDB cluster component to monopolize the corresponding CPU resources, the CPU quota must be an integer greater than or equal to `1` besides setting Guaranteed-level QoS as mentioned above.
```
Comment (xiaojingchen) [inline dev/tidb-in-kubernetes/reference/configuration/tidb-cluster.md]: there have a reference link was omitted.

**Comment by juliezhang1112** [comment]
I create a PR just the same as this one due to CLA failure. See https://github.com/pingcap/docs/pull/1442

**Pull request closed without merging**